CMakeLists.txt

```diff
@@ -316,7 +316,14 @@ set(EXPAT_SHARED_LIBS OFF)
 add_subdirectory(3party/expat/expat)
 add_subdirectory(3party/agg)
 add_subdirectory(3party/bsdiff-courgette)
+
 add_subdirectory(3party/gflags)
+# Not needed for the usual build process, but it fixes QtCreator editor,
+# that doesn't see gflags/gflags.h in binary dir (gflags has tricky cmake configuration).
+if (PLATFORM_DESKTOP)
+  include_directories("${PROJECT_BINARY_DIR}/3party/gflags/include")
```

Comment:
target_include_directories(gflags PUBLIC "${PROJECT_BINARY_DIR}/3party/gflags/include")

Reply:
Tried, but that doesn't work because gflags and gflags::gflags are aliases and target_include_directories doesn't accept it.
Real library name is tricky to fetch from CMakeLists in gflags. An example how to make a simple library use strange build technics for the very simple things.

Reply:
Напиши там туду тогда для меня посмотреть.